src/cmd/graph_pwsh/README.md

```diff
@@ -0,0 +1,103 @@
+# Graph SDK Powershell Troubleshooter
+
+In certain cases, troubleshooting would be significantly simplified if a Corso
+user had a simple mechanism to execute targetted MS Graph API commands against
+their environment.
+
+One convenient mechanism to accomplish this without going down to the level of
+wrapping individual Graph API calls is to use the
+[Microsoft Graph Powershell](https://learn.microsoft.com/en-us/powershell/microsoftgraph/overview?view=graph-powershell-1.0).
+It provides a convenient wrapper and great coverage of the API surface.
+
+## Build container
+
+Before using the tool you want to build the container that packages it.
+
+```sh
+docker build -t corso/graph_pwsh:latest .
```

Comment:
Would love to avoid this step, but unfortunately running into https://github.com/microsoftgraph/msgraph-sdk-powershell/issues/1920 if I try to use the graph SDK published image directly.

Reply:
As of now, this will require the user to pull down the codebase and navigate to this directory. What are your thoughts on publishing an image(granted we will have to publish for all architectures) with `src/cmd/graph_pwsh/Auth-Graph.ps1` included and automatically set to execute on entry?

Reply:
In the case of troubleshooting it might be ok to start with getting the repo. 
Pushing the actual image would be nice.

I did think about folding the auth script in the image, although it was not immediately clear how to have it execute autmatically (needs to be in the same `pwsh` session) but I think it is doable with profiles.

Reply:
Added support for automatic authentication cce261dcc97173e2cc992d870c3c38f569932eea